src/redprl/machine.fun

```diff
@@ -234,7 +234,7 @@ struct
          (case stk of
              [] => raise Final
            | CAP _ :: stk =>
-               STEP @@ cap || (syms, stk)
+               CRITICAL @@ cap || (syms, stk)
```

Comment:
@jonsterling I wonder if there's some way to catch this kind of bugs by tying as in the previous machine.